README.md

```diff
@@ -39,6 +39,23 @@
 
 对于老版本的 LaTeX 发行版，请通过包管理器升级 ctex 的版本。
 
+
+
+此外，对于非Windows环境，会需要额外安装两个字体：
+
+* `Times New Roman`，该字体可以在Windows系统字体库中获取，也可以在Ubuntu/Debian下快速安装
```

Comment:
Then we prefer to install times new roman using apt-get, rather than download it.

Reply:
> 

In CI actions.

Reply:
Use the link to download temporarily, and then change to apt-get if you really need it.